presto-docs/src/main/sphinx/sql/alter-schema.rst

```diff
@@ -21,6 +22,20 @@ Rename schema ``web`` to ``traffic``::
 
     ALTER SCHEMA web RENAME TO traffic
 
+Change owner of schema ``web`` to user ``alice``::
+
+    ALTER SCHEMA web SET AUTHORIZATION alice
+
+Allow everyone to create tables in schema ``web`` (or to drop the schema)::
```

Comment:
```
Allow everyone to drop schema and create tables in schema ``web``::
```
?

Reply:
We're setting authorization to role "PUBLIC" so everyone can create and drop tables and drop the schema itself.
Lemme try to rephrase this in better English.